### ISSUE TITLE
Reporte de um bug na lição Download de páginas Web com Python

### ISSUE THREAD

**Issue by DanielAlvesLABDH** [programminghistorian/jekyll#2506]
@joanacvp reportou um erro no código da lição Download de páginas Web com Python, quando em ambiente Windows. A utilização de "wb" é errada uma vez que estamos a lidar com texto. Deve ser usado "w". Vou corrigir no texto da lição já publicada.

**Comment by anisa-hawes** [comment]
Hello @DanielAlvesLABDH. I'm happy to take care of Lesson Maintenance tasks, so feel free to assign me when something comes up!

The workflow we have implemented is detailed here: <https://github.com/programminghistorian/jekyll/wiki/Lesson-Maintenance-Workflow>

**Comment by anisa-hawes** [comment]
In this case, we'll need to check on the EN, ES and FR versions of the lesson too... This question of `wb` or `w` has come up numerous times, so I'm going to check back through some previous Issues to see what the best way forward is.

**Comment by DanielAlvesLABDH** [comment]
Thanks for the help @anisa-hawes. On Windows changing the "wb" to "w" worked. But we will wait for your suggestion.

**Comment by anisa-hawes** [comment]
No problem. It looks as though that is the decision we came to in Issue #2448 and implemented in PR #2450.

I have just checked and `w` is already used in the EN, ES and FR lessons, so in this case a change in PT will make the code consistent across all:

/en/lessons/working-with-web-pages
/fr/lecons/telecharger-des-pages-web-avec-python
/es/lecciones/trabajar-con-paginas-web
/pt/licoes/download-paginas-web-python

I'll rebase your PR now, and it will be ready to merge.
